bundles/framework/divmanazer/component/SelectList.js

```diff
@@ -82,10 +82,25 @@ Oskari.clazz.define('Oskari.userinterface.component.SelectList',
     update: function() {
       this.element.find('select').trigger('chosen:updated');
     },
-    /**  @method addOption appends a new option to the select
-     *   @param { Object } object with keys id and title
+    /**  
+     * @method enable or disable select
+     * @param { boolean } true = enable
+     */
+    enable: function ( enableFlag ) {
```

Comment:
To be consistent I still would use setEnabled() as for example in https://github.com/nxholm/oskari-frontend/blob/71ad715ace33259dfef51fee2a0f9a4b15e5b86d/bundles/framework/divmanazer/component/FormComponent.js#L70

Reply:
As SelectList extends FormComponent it would have both enable() and setEnabled() so you can just call setEnabled and name this _setEnabledImpl(): https://github.com/nxholm/oskari-frontend/blob/71ad715ace33259dfef51fee2a0f9a4b15e5b86d/bundles/framework/divmanazer/component/FormComponent.js#L86

Reply:
To clarify:
- rename this function _setEnabledImpl
- rename any calls to enable -> setEnabled